eZ/Bundle/EzPublishRestBundle/Resources/config/services.yml

```diff
@@ -369,7 +366,7 @@ services:
 
     ezpublish_rest.input.dispatcher:
         class: %ezpublish_rest.input.dispatcher.class%
-        factory_service: ezpublish_rest.factory
+        factory: ezpublish_rest.factory
```

Comment:
A bit unsure what factory w/o method means, so is this by any chance a left over?